### PR TITLE
Mt 331 transport cards update

### PR DESCRIPTION
**Pull request by lha047**
Relates to MT-331

### CODE REVIEW

File: stories/visualStyle/Typography.stories.svelte [hunk 1]
```diff
@@ -32,10 +32,44 @@
     <p>{paragraph}</p>
     <p class="text-small">{small}</p>
     <form action="">
-      <label for="input">{label}</label>
+      <label for="input" class="form-label">{label}</label>
       <input id="input" class="form-field" type="text" />
     </form>
     <a href="">{link}</a>
+    <a
```
Comment (tajakobsen) [inline stories/visualStyle/Typography.stories.svelte:39]: I'm a bit unsure if this belongs under typography

File: src/scss/global/_links.scss [hunk 2]
```diff
@@ -41,19 +43,22 @@ a {
     }
   }
   &.transport-card {
-    display: flex;
-    flex-direction: column;
+    display: grid;
```
Comment (tajakobsen) [inline src/scss/global/_links.scss:46]: What was the gain in switching to `display: grid`?

**Review by tajakobsen — CHANGES_REQUESTED** [review]
Good job! Just a few small feedback points.

File: stories/visualStyle/Typography.stories.svelte (hunk 1 above)
Comment (lha047) [reply stories/visualStyle/Typography.stories.svelte:39]: I wanted to show the font size of these components. To have a comparison with the other used font sizes

**Review by tajakobsen — APPROVED** [review]
Looks good. Merging to main.

**Pull request merged**